services/catalog/tests/unit/test_package.py

```diff
@@ -23,7 +23,11 @@ def pylintrc(project_slug_dir, osparc_simcore_root_dir):
 
 def test_run_pylint(pylintrc, package_dir):
     cmd = "pylint --jobs 0 --rcfile {} -v {}".format(pylintrc, package_dir)
-    assert subprocess.check_call(cmd.split()) == 0
+    pipes = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
perhaps you can include this code in helpers within ``packages/pytest-simcore``

Reply:
if you have a chance, update as well it in https://github.com/ITISFoundation/cookiecutter-simcore-pyservice as well

Reply:
> perhaps you can include this code in helpers within `packages/pytest-simcore`

Trying to add this at the moment. When I'm done please inspect how the new packages get installed.

Reply:
> if you have a chance, update as well it in https://github.com/ITISFoundation/cookiecutter-simcore-pyservice as well

also trying to apply the same changes here